### PR TITLE
#630 change Fab to button

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
improve warning messages

### CODE REVIEW

**Review by ghost — APPROVED** [review]
The component part seems perfectly fine to me.

File: src/taipy/gui/data/content_accessor.py [hunk 1]
```diff
@@ -58,6 +58,15 @@ def __get_mime_from_file(self, path: pathlib.Path):
                 warnings.warn(f"({path}) cannot read mime type.\n{e}")
         return None
 
+    def __get_display_name(self, var_name: str) -> str:
+        if not isinstance(var_name, str):
+            return var_name
+        if var_name.startswith("_tpC_"):
+            var_name = var_name[5:]
+        if var_name.startswith("tpec_"):
```
Comment (FabienLelaquais) [inline src/taipy/gui/data/content_accessor.py:66]: that would be an 'else', right?

**Review by FabienLelaquais — APPROVED** [review]
Looks just great!

File: src/taipy/gui/data/content_accessor.py (hunk 1 above)
Comment (FredLL-Avaiga) [reply src/taipy/gui/data/content_accessor.py:66]: no, the var can be encapsulated in a TaipyContent holder

Comment (FabienLelaquais) [reply src/taipy/gui/data/content_accessor.py:66]: Oh whao. I actually couldn't figure out where the tpC part came from. Kind of searched... then abandoned.
Makes sense. Thx

**Pull request merged**